src/Configuration/LoggerConfiguration.php

```diff
@@ -76,8 +76,10 @@ function ($exception, $inspector, $run) use ($monolog, $error_handler) {
                             return $result;
                         case $result instanceof ResponseInterface || $result instanceof Redirect:
                             $result->sendContent();
+                            // no break
```

Comment:
It's about the PSR-2 coding standard issue: `There MUST be a comment such as // no break when fall-through is intentional in a non-empty case body.`.